### ISSUE TITLE
Setup intergration tests 

### ISSUE THREAD

**Issue by bokket** [beyondstorage/go-service-hdfs#4]
This is a reference：https://github.com/HariSekhon/Dockerfiles

**Comment by Xuanwo** [comment]
Maybe we need a `setup-hdfs`?

**Comment by bokket** [comment]
> Maybe we need a `setup-hdfs`?

Do we need to rent a cloud service to build this cluster?
There are some unknown pitfalls inside docker:such as privilege level 
Reference：https://github.com/colinmarc/hdfs/issues/263

**Comment by Xuanwo** [comment]
I'm trying to build a `setup-hdfs` here: https://github.com/beyondstorage/setup-hdfs/pull/1

**Comment by Xuanwo** [comment]
Please give `setup-hdfs` a try :smile:

**Comment by Xuanwo** [comment]
Ooops, integration tests on master branch failed. @bokket

**Comment by bokket** [comment]
> Ooops, integration tests on master branch failed. @bokket

```
curl: (7) Failed to connect to 127.0.0.1 port 9000: Connection refused
Error: Process completed with exit code 7.
```
This is puzzling to me.

**Comment by bokket** [comment]
Hi，can we merge and try this pr error on pr #19？

**Comment by Xuanwo** [comment]
Implemented in #24
